components/Projects.js

```diff
@@ -9,41 +9,39 @@ export default function Projects() {
           Projects
         </h1>
       </div>
-      {/* Grid starts here */}
-      <div className="bg-[#F1F1F1] dark:bg-gray-900">
-        <div className="max-w-6xl mx-auto grid grid-cols-1 md:grid-cols-2 gap-8 py-20 pb-40">
-          {userData.projects.map((proj, idx) => (
-            <ProjectCard
-              title={proj.title}
-              link={proj.link}
-              imgUrl={proj.imgUrl}
-              number={`${idx + 1}`}
-            />
-          ))}
-        </div>
+      <div className="bg-[#F1F1F1] dark:bg-gray-900 p-10 grid grid-cols-1 md:grid-cols-2">
+        {userData.projects.map((project, index) => (
+          <ProjectCard projectID={index} />
+        ))}
       </div>
     </section>
   );
 }
 
-const ProjectCard = ({ title, link, imgUrl, number }) => {
+const ProjectCard = ({ projectID }) => {
+  const project = userData.projects[projectID];
+
   return (
-    <a href={link} className="w-full block shadow-2xl">
-      <div className="relative overflow-hidden">
-        <div className="h-72 object-cover">
-          <img
-            src={imgUrl}
-            alt="portfolio"
-            className="transform hover:scale-125 transition duration-2000 ease-out object-cover h-full w-full"
-          />
-        </div>
-        <h1 className="absolute top-10 left-10 text-gray-50 font-bold text-xl bg-red-500 rounded-md px-2">
-          {title}
-        </h1>
-        <h1 className="absolute bottom-10 left-10 text-gray-50 font-bold text-xl">
-          {number.length === 1 ? "0" + number : number}
-        </h1>
+    <div className="rounded flex flex-col justify-between transform hover:scale-105 transition duration-200 shadow-md my-4 mx-4">
+      <img className="w-full shadow-sm" src={project.imgUrl}></img>
+      <div className="px-6 py-4 justify-self-start">
+        <div className="font-bold text-xl mb-2">{project.title}</div>
+        <p className="text-gray-700 text-base">{project.description}</p>
+      </div>
+      <div className="grid grid-cols-2">
+        <a
+          href={project.gitHubUrl}
+          className="bg-gray-200 hover:bg-gray-300 px-3 py-1 border border-gray-400 rounded-sm text-sm font-semibold text-gray-700 my-2 ml-3 mr-1"
+        >
+          GitHub
+        </a>
+        <a
+          href={project.siteUrl}
+          className="bg-gray-200 hover:bg-gray-300 px-3 py-1 border border-gray-400 rounded-sm text-sm font-semibold text-gray-700 my-2 ml-1 mr-3"
+        >
+          Live Site
+        </a>
```

Comment:
Not sure about what the best way to do these buttons with external links is, they're simple and functional now but I would like to make them look a bit more pleasing.

Reply:
<img width="602" alt="image" src="https://user-images.githubusercontent.com/6580472/155043052-f633cb4f-7617-4d70-bb78-4bfeac59f582.png">

Reply:
I hear ya! Took a stab at prettying them up - see my draft PR